.github/workflows/test-cron.yaml

```diff
@@ -1,38 +1,50 @@
 name: Alloy-Test-Cron
 on:
+  repository_dispatch:
+    types: run-e2e
   schedule:
     - cron: "0 */24 * * *"
 
 jobs:
   alloy-prod-e2e:
-    name: "Prod E2E Tests"
+    name: "Cron: Prod E2E Tests"
     runs-on: ubuntu-latest
+    outputs:
+      previous-tag: ${{ steps.previoustag.outputs.tag }}
     steps:
-    - uses: actions/checkout@v2
-      with:
-        ref: "main"
-    - uses: actions/cache@v2
-      id: npm-cache
-      with:
-        path: '**/node_modules'
-        key: ${{ runner.os }}-modules-${{ hashFiles('**/package-lock.json') }}
-    - name: Install dependencies
-      if: steps.npm-cache.outputs.cache-hit != 'true'
-      run: npm ci
-    - name: 'BrowserStack Env Setup'
-      uses: 'browserstack/github-actions/setup-env@master'
-      with:
-        username:  ${{ secrets.BROWSERSTACK_USERNAME }}
-        access-key: ${{ secrets.BROWSERSTACK_ACCESS_KEY }}
-    - name: 'BrowserStackLocal Setup'
-      uses: 'browserstack/github-actions/setup-local@master'
-      with:
-        local-testing: start
-        local-identifier: random
-    - name: Run TestCafe Tests
-      run: |
-        npm run test:functional:cron:prod
-    - name: 'BrowserStackLocal Stop'
-      uses: 'browserstack/github-actions/setup-local@master'
-      with:
-        local-testing: stop
\ No newline at end of file
+      - uses: actions/checkout@v2.3.3
+        with:
+          fetch-depth: 0
+      - name: Get previous tag
+        id: previoustag
+        uses: "WyriHaximus/github-action-get-previous-tag@v1"
```

Comment:
Could this potentially find a candidate release tag? We need to be careful when we create the tag for a release so that its not pulled in here before it is ready.